.gitignore

```diff
@@ -294,5 +294,5 @@ __pycache__/
 # Functions Click
 functions-extensions/
 
-.vscode/
+
```

Comment:
I think this change was accidental. Can we revert it?

Reply:
Sure